docs/source/concept_models/vivarium_iv_iron/maternal_model/concept_model.rst

```diff
@@ -403,17 +403,19 @@ Probability of sampling from a given country's hemoglobin distribution using the
     - Value
     - Note
   * - :math:`population_\text{country}`
-    - Population size of pregnant and lacting women for a given national location
-    - See definition of PLW in table above
+    - Population size of women of reproductive age for a given national location
+    - See definition of WRA in table above
     - Summed across all age groups
   * - :math:`population_\text{region}`
-    - Population size of pregnant and lactating women for a given regional location
+    - Population size of women of reproductive age for a given regional location
     - :math:`\sum_{country=1}^{n} population_\text{country}`
     - For all countries within the region
 
-.. note::
+.. note:: 
+
+  We may update the weighting unit to pregnant and lacting women (PLW) rather than WRA once we have confirmed that our hemoglobin exposure model is functioing properly by validating to GBD. Although the hemoglobin exposure distribution is specific to women of reproductive age, we have chosen to weight the hemoglobin exposure distribution to the population size of pregnant and lactating women due to our explicit project focus on PLW. This prioritization of hemoglobin exposures among PLW may cause slight differences in our location-aggregated esimates of anemia among non-pregnant or postpartum simulants among WRA compared to the regional estimates from GBD.
```

Comment:
```suggestion
  We may update the weighting unit to pregnant and lactating women (PLW) rather than WRA once we have confirmed that our hemoglobin exposure model is functioning properly by validating to GBD. Although the hemoglobin exposure distribution is specific to women of reproductive age, we have chosen to weight the hemoglobin exposure distribution to the population size of pregnant and lactating women due to our explicit project focus on PLW. This prioritization of hemoglobin exposures among PLW may cause slight differences in our location-aggregated estimates of anemia among non-pregnant or postpartum simulants among WRA compared to the regional estimates from GBD.
```